tests/32bit.conf

```diff
@@ -0,0 +1,19 @@
+COMPILE | -DAPP=ATM -DCCPP_SUITES=FV3_RAP,FV3_HRRR -D32BIT=ON -DCCPP_32BIT=ON                                                     |                | fv3 |
```

Comment:
Since these tests are added in rt.conf/rt_gnu.conf, would you please remove the two 32bit.conf and 32bit_gnu.conf?

Reply:
If you read my earlier comment, I explain those files:

> The 32bit.conf and 32bit_gnu.conf files should be removed before this is merged. They're in the PR now, just so the testers know what tests need new baselines.